controller/common/node_info.py

```diff
@@ -52,5 +55,16 @@ def is_array_iscsi_iqns_match(self, host_iqns):
         host_iqns_lower = [iqn.lower() for iqn in host_iqns]
         return self._iscsi_iqn_lowercase in host_iqns_lower
 
+    def is_array_nvme_nqn_match(self, host_nqns):
+        """
+        Args:
+           host_nqns: storage host nqns list
+
+        Returns:
+           Is current host nqns matches
+        """
+        host_nqns_lower = [nqn.lower() for nqn in host_nqns]
+        return self._nvme_nqn_lowercase in host_nqns_lower
+
     def __str__(self):
-        return "iscsi_iqn: " + self.iscsi_iqn + ", fc_wwns: " + ",".join(self.fc_wwns)
+        return "nvme_nqn: " + self.nvme_nqn + ", fc_wwns: " + ",".join(self.fc_wwns) + " iscsi_iqn: " + self.iscsi_iqn
```

Comment:
not a must, but since it's starting to get long, I think format would be more readable

Reply:
done